Makefile

```diff
@@ -35,3 +39,11 @@ protobuf:
 cover:
 	go test ./pkg/... -coverprofile=coverage.out
 	go tool cover -html=coverage.out
+
+release:
+	@if [ ! -z "$(GITHUB_TOKEN)" ];then \
```

Comment:
Will it work for anyone's GITHUB_TOKEN?

Reply:
@sana-jawad you need to have salesforce github token to push releases to salesforce repository. If we are unable to to get that token we skip artifact publish and just push the docker image.